server/src/use-cases/version-control/azure-devops/helpers/date.helpers.unit.test.js

```diff
@@ -0,0 +1,29 @@
+import { describe, it, expect } from '@jest/globals';
+import { getGmtISOString, getNextDayGmtISOString } from './date.helpers.js';
+
+// Mock the global Date object
+global.Date = class extends Date {
```

Comment:
@sathish-thangavel is this approach a valid approach Sathish?

Reply:
Yes since we need base reference for time this approach works